.storybook/main.js

```diff
@@ -83,12 +83,28 @@ module.exports = /** @type {Omit<StorybookConfig,'typescript'|'babel'>} */ ({
       },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30633&commitId=cf9aef1e4f4eb499ab97a92f38a4585333bd1d98&env=prod&clientType=FLUENTUI)


  <details><summary><b>GroupedList</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|GroupedList.Root.click.chromium.png|0| Removed|
|GroupedList.Root.default.chromium.png|0| Removed|
|GroupedList.Root.hover.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>